tests/phpunit/tests/dependencies/scripts.php

```diff
@@ -2023,8 +2141,8 @@ public function test_wp_add_inline_script_after_with_concat_and_conditional_and_
 
 		$expected  = "<script type='text/javascript' src='/wp-admin/load-scripts.php?c=0&amp;load%5Bchunk_0%5D=jquery-core,jquery-migrate&amp;ver={$wp_version}'></script>\n";
 		$expected .= "<!--[if gte IE 9]>\n";
-		$expected .= "<script type='text/javascript' src='http://example.com' id='test-example-js'></script>\n";
-		$expected .= "<script type='text/javascript' id='test-example-js-after'>\nconsole.log(\"after\");\n</script>\n";
+		$expected .= "<script type=\"text/javascript\" src=\"http://example.com\" id=\"test-example-js\"></script>\n";
+		$expected .= "<script type=\"text/javascript\" id=\"test-example-js-after\">\n/* <![CDATA[ */\nconsole.log(\"after\");\n/* ]]> */\n</script>\n";
```

Comment:
There are a few cases like this where the contents of an (IE conditional) _HTML comment_ were changed.

These script tags, on their own, would be recognized as equivalent with double `"` or single `'` quotes. But they're not script tags, they are the text contents of an HTML comment where `'` is not the same as `"`.